zebra-network/src/protocol/external/codec.rs

```diff
@@ -72,6 +76,13 @@ impl Builder {
         }
     }
 
+    /// Configure the codec for the given peer address.
+    #[allow(dead_code)]
+    pub fn for_address(mut self, addr: SocketAddr) -> Self {
+        self.addr = Some(addr);
+        self
+    }
```

Comment:
This way we can set it to `None` if we want to unset an address while still letting us pass a `SocketAddr` directly.

```suggestion
    pub fn for_address(mut self, addr: impl Into<Option<SocketAddr>>) -> Self {
        self.addr = addr.into();
        self
    }
```